src/components/keyboard/index.jsx

```diff
@@ -26,11 +31,57 @@ export default function Keyboard({ word, updateActive, checkGuess, gameOver }) {
         }
     }
 
+    function handleClick(event) {
+        console.log(event);
+        if (event === "BACK") {
+            const newInput = input.slice(0, -1);
+            setInput(newInput);
+            updateActive(newInput);
+            return;
+        }
+
+        if (event === "ENTER") {
+            handleSubmit(null);
+            return;
+        }
+
+        const newInput = input + event;
+        if (newInput.length <= word.length) {
+            setInput(newInput);
+            updateActive(newInput);
+        }
+    }
+
+    function KeyboardRow({ rowLayout }) {
+        return (
+            <div className="keyboard-row">
+                {rowLayout && rowLayout.toUpperCase().split('').map((letter, idx) =>
```

Comment:
Rather than using `toUpperCase()` here, why not just store it as upper case? Will we ever need it lower case?

Reply:
Oh thats a good point yeah lemme take a look